File: Source/Rio.Web/src/app/shared/models/offer/trade-dto.ts
import { TradeStatusDto } from './trade-status-dto';
import { PostingDto } from '../posting/posting-dto';
import { AccountDto } from '../account/account-dto';

export class TradeDto {
    TradeID: number;
    TradeNumber: string;
    CreateAccount: AccountDto;
    TradeStatus: TradeStatusDto;
    Posting: PostingDto;

    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}

File: Source/Rio.Models/DataTransferObjects/Offer/TradeDto.cs
using Rio.Models.DataTransferObjects.Account;
using Rio.Models.DataTransferObjects.Posting;

namespace Rio.Models.DataTransferObjects.Offer
{
    public class TradeDto
    {
        public int TradeID { get; set; }
        public string TradeNumber { get; set; }
        public AccountDto CreateAccount { get; set; }
        public TradeStatusDto TradeStatus { get; set; }
        public PostingDto Posting { get; set; }
    }
}